keyboards/planck/keymaps/ipetepete/keymap.c

```diff
@@ -0,0 +1,243 @@
+  {_______,   KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______},
+  {_______, _______, KC_LCBR, KC_LBRC, KC_LPRN, KC_PIPE, KC_BSLS, KC_LPRN, KC_LBRC, KC_LCBR, _______, _______},
+  {_______, _______, _______, _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY}
+},
+
+[_RAISE] = {
+  {KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_MINS, KC_EQL,  KC_BSPC},
+  {KC_GRV,   KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______},
+  {_______, _______, _______, LGUI(KC_C), LGUI(KC_V),       KC_LPRN, KC_RPRN, KC_1,  KC_2,  KC_3, KC_ASTR, _______},
+  {_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______}
+},
+
+[_ADJUST] = {
+  {RESET  , _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______},
+  {_______, _______, _______, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, M_QWERTY,M_GAMR,   _______,  _______,  _______},
+  {_______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  _______, _______, _______, _______, _______},
+  {_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ }
+},
+
+[_FUNC] = {
+  {RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   XXXXXXX, KC_7,  KC_8,  KC_9, KC_PLUS,  VOLUP  },
+  {XXXXXXX, KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  XXXXXXX, KC_4,  KC_5,  KC_6, KC_MINS,  VOLDN  },
+  {KC_CAPS, KC_F11,  KC_F12,  KC_F13,  KC_F14,  KC_F15,  KC_0,    KC_1,  KC_2,  KC_3, KC_ASTR,  KC_MUTE},
+  {_______, KC_LSFT, _______, POWER , KC_MPRV, KC_MPLY, KC_MPLY, KC_MNXT, HYPER , _______, KC_RSFT, _______}
+},
+
+
+[_SPCFN] = {
+  {KC_TILD, _______, KC_HOME, KC_PGUP, KC_END , KC_LBRC, KC_RBRC, SRTLIN , KC_UP  , ENDLIN , KC_PGUP, KC_DEL  },
+  {KC_GRV,  _______, SPCLFT , KC_PGDN, SPCRGT , KC_LCBR, KC_RCBR, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, _______ },
+  {_______, _______, _______, _______, _______, KC_LPRN, KC_RPRN, _______, _______, _______, _______, _______ },
+  {_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, BL_TOGG, BL_STEP }
+},
+
+/* Mouse layer */
+[_MOUS] = {
+  {_______, KC_ACL2, KC_ACL1, KC_ACL0,  XXXXXXX, XXXXXXX, XXXXXXX,  XXXXXXX, KC_MS_U, XXXXXXX, XXXXXXX, XXXXXXX },
+  {_______, XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  KC_MS_L, KC_MS_D, KC_MS_R, XXXXXXX, XXXXXXX },
+  {XXXXXXX, XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX },
+  {XXXXXXX, XXXXXXX, XXXXXXX,  XXXXXXX, KC_BTN1, _______, _______, KC_BTN2, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX }
+}
+
+
+};
+
+
+void persistant_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+const uint16_t PROGMEM fn_actions[] = {
+  [0] = ACTION_MODS_ONESHOT(MOD_RSFT),
+};
+
```

Comment:
Both Action Functions and Action Macros have been deprecated. 

This can be accomplished with just the `OSM(MOD_RSFT)` keycode, now.